doc/flatpak-run.xml

```diff
@@ -310,7 +310,7 @@
                     Don't expose a well known socket to the application. This overrides to
                     the Context section from the application metadata.
                     <arg choice="plain">SOCKET</arg> must be one of: x11, wayland, fallback-x11, pulseaudio, system-bus, session-bus,
-                    ssh-auth, pcsc, cups.
+                    a11y-bus, ssh-auth, pcsc, cups.
```

Comment:
Similar additions need to be done in the other docs that have lists like this. Grep for e.g. cups to see them.